Metro-Records/la-metro-councilmatic
# Update board member map

## Overview

This PR:

- Adds Caltrans as an administrative layer to the board members map
- Fixes the data table header
- Renames District to Role in table header
- Matches data table labels to toggles above map (connects #1020)
- Fixes bug where no member at the LA administrative level apart from the mayor appears in the info box on data table hover

Connects #1028

### Demo

![member map](https://github.com/Metro-Records/la-metro-councilmatic/assets/12176173/89e9a4e9-e9f4-41fb-b844-15e602348651)

## Testing Instructions

Please test using the staging site: https://lametro.datamade.us

 * View the board member map and table and confirm it looks good
 * Scroll the data table and confirm the header remains fixed
 * Confirm the toggles match the labels in the data table
 * Confirm the correct polygon is highlighted and the info box populates for each member when hovering over the data table
